### PR TITLE
#16 Move Profile in to esn-frontend-account

### PR DESCRIPTION
**Pull request by fabienmoyon**
## Desktop

OpenPaas (1) (1)

## Mobile

localhost_9900_(iPhone X) (1)

### CODE REVIEW

File: src/linagora.esn.controlcenter/app/profile-menu-controlcenter/profile-menu-controlcenter.pug [hunk 1]
```diff
@@ -1,4 +1,4 @@
 md-menu-item
   md-button(ui-sref="controlcenter")
     md-icon(md-svg-icon="account-settings-variant",aria-label="account-settings-variant Icon")
-    | {{ 'Control Center' | translate }}	
\ No newline at end of file
+    | {{ 'Account' | translate }}
```
Comment (rezk2ll) [inline src/linagora.esn.controlcenter/app/profile-menu-controlcenter/profile-menu-controlcenter.pug:4]:
```suggestion
    | {{ 'Account' | translate }}

```

File: src/linagora.esn.profile/app/sidebar/sidebar-menu-item.component.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+(function(angular) {
```
Comment (rezk2ll) [inline src/linagora.esn.profile/app/sidebar/sidebar-menu-item.component.js:1]: IIFE are no longer needed i guess

File: src/linagora.esn.profile/app/sidebar/sidebar.less [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+  hr {
+    margin: 10px 0;
+  }
+
+  .listview a.lv-item:hover {
+    color: @secondaryTextColor;
+    background-color: @darkenBgColor;
+  }
+}
```
Comment (rezk2ll) [inline src/linagora.esn.profile/app/sidebar/sidebar.less:57]:
```suggestion
}

```

**Review by asmaNode — APPROVED** [review]

**Review by asmaNode — APPROVED** [review]

**Comment by tuanlc** [comment]
From your demo on desktop, when you choose `profil` item in the sidebar, it is not highlighted (Darker) like when you choose `Parametre`. Do you see it?

File: src/linagora.esn.profile/app/sidebar/sidebar-menu-item.component.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+
```
Comment (tuanlc) [inline src/linagora.esn.profile/app/sidebar/sidebar-menu-item.component.js:1]: remove this blank line

File: src/linagora.esn.profile/app/sidebar/sidebar.component.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+(function(angular) {
```
Comment (tuanlc) [inline src/linagora.esn.profile/app/sidebar/sidebar.component.js:1]: No need IIFE anymore

File: src/linagora.esn.profile/app/sidebar/sidebar-menu-item.pug [hunk 6]
```diff
@@ -0,0 +1,3 @@
+a.lv-item.menu-item(ng-href="{{::$ctrl.href}}")
+  i.mdi(ng-class="::$ctrl.icon")
+  .menu-title {{::$ctrl.label | esnI18n}}
```
Comment (tuanlc) [inline src/linagora.esn.profile/app/sidebar/sidebar-menu-item.pug:3]: Does it work? Should be:

```suggestion
  .menu-title {{::$ctrl.label | translate}}
```

**Review by tuanlc — CHANGES_REQUESTED** [review]

File: src/linagora.esn.profile/app/app.less [hunk 7]
```diff
@@ -127,9 +131,16 @@
...
     align-items: stretch;
     flex-direction: column;
-    .po-card-image img {
+    .po-card-image {
+      border-radius: 50%;
+      overflow: hidden;
+      display: block;
+      width: 60px;
+      height: 60px;
+     img {
```
Comment (huy-ta) [inline src/linagora.esn.profile/app/app.less:140]: Wrong indentation.

File: src/linagora.esn.controlcenter/app/profile-menu-controlcenter/profile-menu-controlcenter.pug [hunk 8]
```diff
@@ -1,4 +1,4 @@
...
```
Comment (huy-ta) [inline src/linagora.esn.controlcenter/app/profile-menu-controlcenter/profile-menu-controlcenter.pug:4]: Please add a blank line at the end of the file here.

**Comment by fabienmoyon** [comment]
> From your demo on desktop, when you choose `profil` item in the sidebar, it is not highlighted (Darker) like when you choose `Parametre`. Do you see it?

Yes it's fixed ;)

**Review by chamerling — APPROVED** [review]

**Pull request merged**